grails-app/views/eventResultDashboard/showAll.gsp

```diff
@@ -292,5 +292,8 @@
     </asset:script>
 </content>
 
+<sitemesh:parameter name="needsAngular" value="true"/>
```

Comment:
These can be deleted so the component in itself is not references while it is not final.